### ISSUE TITLE
[amplify_secure_storage_dart] Support `win32: ^5.0.2`

### ISSUE THREAD

**Issue by asaarnak** [aws-amplify/amplify-flutter#3114]
Version solving conflict with file_picker dependency.
```
Because no versions of amplify_auth_cognito match >1.1.0 <2.0.0 and amplify_auth_cognito 1.1.0 depends on amplify_analytics_pinpoint_dart ^0.3.0, amplify_auth_cognito ^1.1.0 requires amplify_analytics_pinpoint_dart ^0.3.0.
And because no versions of amplify_analytics_pinpoint_dart match >0.3.0 <0.4.0 and amplify_analytics_pinpoint_dart 0.3.0 depends on amplify_secure_storage_dart ^0.4.0, amplify_auth_cognito ^1.1.0 requires amplify_secure_storage_dart ^0.4.0.
Because amplify_secure_storage_dart 0.4.0 depends on win32 ^4.1.2 and no versions of amplify_secure_storage_dart match >0.4.0 <0.5.0, amplify_secure_storage_dart ^0.4.0 requires win32 ^4.1.2.
Thus, amplify_auth_cognito ^1.1.0 requires win32 ^4.1.2.
And because file_picker >=5.3.2 depends on win32 ^5.0.2, amplify_auth_cognito ^1.1.0 is incompatible with file_picker >=5.3.2.
So, because my_app depends on both amplify_auth_cognito ^1.1.0 and file_picker ^5.3.2, version solving failed.

**Comment by dnys1** [comment]
Hi @asaarnak, thanks for the alert. You've opened a number of these and I apologize that we haven't caught these ahead of time. I'll look into ways we can gain better awareness of our integration with the larger ecosystem.

In the meantime, I've opened #3115 to resolve this and you're welcome to set the following dependency override while we await release of the fix:

```yaml
dependency_overrides:
  win32: ">=4.1.2 <6.0.0"
```

**Comment by dnys1** [comment]
This has been resolved in Amplify Flutter v1.1.1 and we have added CI checks for catching integration issues with the larger pub ecosystem.
